pkgs/jni/lib/src/jobject.dart

```diff
@@ -125,22 +137,36 @@ class JObject {
     reference.release();
   }
 
+  /// Whether this object is of the given [type].
+  ///
+  /// For example:
+  ///
+  /// ```dart
+  /// if (object.isA(JLong.type)) {
+  ///   final i = object.as(JLong.type).longValue;
+  ///   ...
+  /// }
+  /// ```
+  bool isA<T extends JObject?>(JObjType<T> type) {
```

Comment:
I like that the name is similar to package:test's 👍

Reply:
;-)